gramine_ratls.go

```diff
@@ -106,19 +106,21 @@ import (
 	"encoding/pem"
 	"errors"
 	"fmt"
+	"os"
 	"unsafe"
 
-	"github.com/sirupsen/logrus"
```

Comment:
Can you revert to logrus? You were right - with the official log package not supporting log levels, it doesn't make much sense to use it. logrus is beeing used by Docker so I'm confident it is being monitored closely for security issues.